### PR TITLE
updating a typo :-)

### PR DESCRIPTION
**Pull request by abhi-11nav**
Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Review by albanD — CHANGES_REQUESTED** [review]
This is already in the loss function: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html 
So we don't need it I think.
